scripts/README.md

```diff
@@ -0,0 +1,29 @@
+# Scripts for generating figures and tables
+
+This folder contains all scripts used to generate figures and tables.
+
+## Figure set up
+
+The `figure_setup` folder contains scripts that are required to be run _prior to_ generating figures.
+
+1. `sync-metadata.R`: This script is used to sync any metadata files found on S3 to a local folder.
+In particular, the `scpca-sample-metadata.tsv` and `scpca-library-metadata.tsv` will be stored to a folder within the root directory of this repo named `s3_files`.
+In order to generate some of the figures (see more on which figures require this script below), this script will be need to run first.
```

Comment:
Worth also mentioning that while the directory is placed in the root, it's also ignored so it won't be tracked with the repo. This means the script _has_ to be run; you can't just get these files by cloning the repo and/or pulling down. You do say it needs to be run first, but since you also say it's in the repo root, I think it's worth clarifying a bit more.